src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -84,33 +84,27 @@ def test_node_coloured_name():
     # Check that we can change the name of the Node and the colour associated
     # with it
     tnode._text_name = "ATest"
-    tnode._colour_key = "Schedule"
+    tnode._colour = "white"
     assert tnode.coloured_name(False) == "ATest"
-    assert tnode.coloured_name(True) == colored(
-        "ATest", SCHEDULE_COLOUR_MAP["Schedule"])
-    # Check that an unrecognised colour-map entry gives us un-coloured text
-    tnode._colour_key = "not-recognised"
-    assert tnode.coloured_name(True) == "ATest"
+    assert tnode.coloured_name(True) == colored("ATest", "white")
```

Comment:
Probably need to put back/modify the old part of the test for _colour = None. What happens if a developer accidentally mis-spells the name of the colour? I assume we get a reasonable stack-trace in that case but possibly we should catch it. In fact, this relates to a slight concern that I have about this change in that we've gone from having `termcolor`-specific settings (i.e. the names of the colours) in a single location to having them scattered through the source.

Reply:
Good point. I've added a check for an invalid colour and an associated test.